examples/nuxt/app.vue

```diff
@@ -0,0 +1,21 @@
+<script setup lang="ts">
+const { $trpc } = useNuxtApp();
+// 💡 Tip: CMD+Click (or CTRL+Click) on `greeting` to go to the server definition
+const { data } = $trpc.greeting.useQuery({ name: 'tRPC user' });
```

Comment:
We should make a `@trpc/tanstack-vue-query` with the same semantics as the new react package 👀👀